rsconf/component/docker.py

```diff
@@ -78,6 +79,13 @@ def internal_build(self):
             run_d=z.run_d,
         )
 
+    def _check_min_version(self, z):
+        z.setdefault("min_version", "0.0.0")
+        p = [int(v) for v in z.min_version.split(".")]
+        if len(p) != 3:
+            raise ValueError(f"min_version={z.min_version} needs to be in X.X.X format")
+        z.min_version = p
```

Comment:
This is not good practice. Do not overwrite a configuration value with a different type. See my discussion below about `_version_parts`.

Reply:
0349609